federation/manifests/federation-ns.yaml

```diff
@@ -0,0 +1,4 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: {{.FEDERATED_NAMESPACE}}
```

Comment:
hmm. So how do I use these manifests to bring up the federation control plane (for developing or setting up federation)?
I was hoping I can just do `kubectl create -f federation/manifests` but clearly that is not going to be enough. I guess we will need a script that will first replace the env vars by their values and then call kubectl create?